src/modules/client/components/planning/EventHistoryFeed.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="history-container" :style="{ height: `${height}px`, top: `${this.top}px` }">
-    <div class="row history-title">
+    <div class="row history-container-title">
```

Comment:
Pourquoi `this.top` et pas juste `top` ?

Reply:
tu pux corriger en top